app/models/product.rb

```diff
@@ -12,8 +12,8 @@ class Product < ApplicationRecord
 
 
   enum gender: {
-    "female": 0,
-    "male": 1
+    "Female": 0,
+    "Male": 1
```

Comment:
Hahaha no worries. I had to do this also because the simple form dropdown needs to be accurate and in line with the attribute value.